### ISSUE TITLE
FAILED: InputStreamSinkSpec

### ISSUE THREAD

**Issue by drewhk** [akka/akka#19585]
https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/4977/consoleFull

```
[info] - must throw exception when call read with wrong parameters *** FAILED *** (18 milliseconds)
[info]   java.lang.IllegalStateException: not yet initialized: only setHandler is allowed in GraphStageLogic constructor
```

**Comment by agolubev** [comment]
I'm taking this. Just need to add CallbackWrapper or similar mechanism to InputStreamSink

**Comment by drewhk** [comment]
fixed in #19625 
(still needs backport)
